components/email/src/Email.svelte

```diff
@@ -1609,37 +1662,47 @@
                           </div>
                         </div>
                         <div class="message-to">
-                          {#each message?.to.slice(0, PARTICIPANTS_TO_TRUNCATE) as to, i}
-                            <div>
-                              <span>
-                                {i === 0 ? "to " : ""}
-                                {#if _this.you && to?.email === _this.you.email_address}
-                                  Me
+                          {#await getAllRecipients( { to: message.to, cc: message.cc, bcc: message.bcc }, ) then allRecipients}
+                            {#each allRecipients.slice(0, PARTICIPANTS_TO_TRUNCATE) as recipient, i}
+                              <p>
+                                {#if i === 0}
+                                  {`to ${
+                                    _this.you &&
+                                    recipient.email === _this.you.email_address
+                                      ? "Me"
+                                      : ""
+                                  }`}
+                                {:else if recipient._type === "cc" && i === message.to.length}
+                                  cc:
+                                {:else if recipient._type === "bcc" && i === message.to.length + message.cc.length}
+                                  bcc:
                                 {/if}
-                                {#if to.email && to.name}
-                                  {to.name ?? _this.you.name} &lt;{to.email}&gt;
-                                {:else if to.email && !to.name}
-                                  {to.email}
+
+                                {#if recipient.email && recipient.name}
+                                  {recipient.name ?? _this.you.name} &lt;{recipient.email}&gt;
+                                {:else if recipient.email && !recipient.name}
+                                  {recipient.email}
                                 {/if}
```

Comment:
Unfortunately, I don't see a way of simplifying this logic until we have child svelte components.